src/com/lftechnology/java_training/jan7/FileInput.java

```diff
@@ -0,0 +1,134 @@
+package com.lftechnology.java_training.jan7;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.util.Scanner;
+import java.util.logging.Level;
+
+/**
+ * Write a program called FileInput to read an int, a double, and a String form a text file called "in.txt", and produce the following
+ * output: (Hint use Scanner to read from file) a. The integer read is 12 b. The floating point number read is 33.44 c. The String read is
+ * "Peter" d. Hi! Peter, the sum of 12 and 33.44 is 45.44
+ * 
+ * @author UtsabNeupane<utsabneupane@lftechnology.com>
+ * 
+ */
+public class FileInput {
+	private static int[] intReadValue = new int[10];
+	private static double[] doubleReadValue = new double[10];
+	private static String[] stringReadValue = new String[10];
+	private static int intValueCounter = 0;
+	private static int doubleValueCounter = 0;
+	private static int stringValueCounter = 0;
+	private static TrainingConstantsUtility constantValues = new TrainingConstantsUtility();
+
+	public static void main(String[] args) {
+		// read values from file
+		readFromFile();
+		// display values written in file to console
+		displayValueFromFile();
+	}
+
+	/**
+	 * read data from file and store it according to its type
+	 */
+	private static void readFromFile() {
+		Scanner inputFile = null;
+
+		try {
+			inputFile = new Scanner(new BufferedReader(new FileReader("in.txt")));
```

Comment:
Use constant for fixed file location, this file might be used in multiple places